aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -20,6 +20,14 @@
       "maxLength": 255,
       "pattern": "^[\\w-]+$"
     },
+    "ComputePlatform": {
+      "description": "The compute platform of the profiling group.",
+      "type": "string",
+      "enum": [
+        "Default",
```

Comment:
As mentioned in the CreateProfilingGroup request doc : https://docs.aws.amazon.com/codeguru/latest/profiler-api/API_CreateProfilingGroup.html#profiler-CreateProfilingGroup-request-computePlatform

Valid values for `ComputePlatform` are :
* Default
* AWSLambda

And hence when we create a profiling group via CLI it would accept these two values for the _ComputePlatform_. Similarly, the CloudFormation should support these valid values.

Whereas having _Others_ on the console is a UX choice decided during the review.

Reply:
> It looks like the UI offers the _Lambda_ and _Other_ options for the compute platform, while the CloudFormation offers _Lambda_ and _Default_:
> 
> * https://docs.aws.amazon.com/codeguru/latest/profiler-ug/setting-up.html#setting-up-step-2
> * https://docs.aws.amazon.com/codeguru/latest/profiler-api/API_CreateProfilingGroup.html#profiler-CreateProfilingGroup-request-computePlatform
> 
> As part of this PR, it looks we'll keep the _Default_ to be consistent, but maybe we should consider updating the UI to be consistent.

As mentioned in the CreateProfilingGroup request doc : https://docs.aws.amazon.com/codeguru/latest/profiler-api/API_CreateProfilingGroup.html#profiler-CreateProfilingGroup-request-computePlatform

Valid values for `ComputePlatform` are :
* Default
* AWSLambda

And hence when we create a profiling group via CLI it would accept these two values for the _ComputePlatform_. Similarly, the CloudFormation should support these valid values.

Whereas having _Others_ on the console is a UX choice decided during the review.